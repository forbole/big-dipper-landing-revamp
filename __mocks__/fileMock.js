// __mocks__/fileMock.js
module.exports = {
  blurDataURL: 'data:image/png;base64,imagedata',
  height: 24,
  src: '/img.jpg',
  width: 24,
}