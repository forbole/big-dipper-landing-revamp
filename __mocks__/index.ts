export { default as MockTheme } from "./theme";
