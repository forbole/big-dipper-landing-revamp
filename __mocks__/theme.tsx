/* eslint-disable */
import { ThemeProvider } from '@mui/material/styles';
import { ReactNode } from 'react';
import { darkTheme } from '@/src/styles/theme';

/**
 * Theme mocker to handle custom keys
 */
const MockTheme = ({ children }: { children: ReactNode }) => {
  return <ThemeProvider theme={darkTheme}>{children}</ThemeProvider>;
};

export default MockTheme;
