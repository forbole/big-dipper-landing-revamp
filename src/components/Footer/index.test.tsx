import { MockTheme } from "@/__mocks__";
import { render, screen } from "@testing-library/react";
import React from "react";

import Component from ".";

// ==================================
// unit tests
// ==================================
describe("Footer", () => {
  it("matches snapshot", async () => {
    const { container } = render(
      <MockTheme>
        <Component />
      </MockTheme>,
    );

    expect(screen.getByText(/productOf/i)).toBeInTheDocument();
    expect(screen.getByText(/privacyPolicy/i)).toBeInTheDocument();
    expect(container.querySelector("footer")).not.toBeNull();
    expect(container).toMatchSnapshot();
  });

  afterEach(() => {
    jest.clearAllMocks();
  });
});
