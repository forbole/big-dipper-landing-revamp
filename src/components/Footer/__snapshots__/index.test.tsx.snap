// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
.emotion-0 .footer__bottom {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-flow: row nowrap;
  -webkit-flex-flow: row nowrap;
  -ms-flex-flow: row nowrap;
  flex-flow: row nowrap;
  gap: 40px;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  justify-item: center;
  max-width: 1024px;
  padding: 0px 24px;
  width: 100%;
}

@media (max-width:767.95px) {
  .emotion-0 .footer__bottom {
    -webkit-box-flex-flow: row wrap;
    -webkit-flex-flow: row wrap;
    -ms-flex-flow: row wrap;
    flex-flow: row wrap;
  }
}

.emotion-0 .footer__icons-wrapper {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-box-flex-wrap: nowrap;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  gap: 32px;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  min-width: 300px;
  padding: 0px 48px;
  width: 100%;
}

@media (min-width:768px) {
  .emotion-0 .footer__icons-wrapper {
    gap: 8px;
    -webkit-box-pack: start!important;
    -ms-flex-pack: start!important;
    -webkit-justify-content: flex-start!important;
    justify-content: flex-start!important;
    margin-bottom: 0;
    margin-right: -128px;
    padding: 0;
    padding-left: 0;
    width: auto;
  }
}

.emotion-0 .footer__icons-wrapper path {
  fill: #fff;
}

.emotion-0 .footer__scroll-top {
  -webkit-align-content: flex-end;
  -ms-flex-line-pack: flex-end;
  align-content: flex-end;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: url(/images/background/footerXs.svg);
  -webkit-background-position: 50% 0%;
  background-position: 50% 0%;
  background-repeat: no-repeat;
  -webkit-background-size: cover;
  background-size: cover;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-flow: column nowrap;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-box-pack: end;
  -ms-flex-pack: end;
  -webkit-justify-content: flex-end;
  justify-content: flex-end;
  min-height: 226px;
  padding: 30px 0px 16px 0px;
  position: relative;
  text-align: center;
}

.emotion-0 .footer__scroll-top .MuiFab-root {
  background-color: rgba(255, 255, 255, 0.2);
  border: 1px solid #EFEFEF;
  border-radius: 50%;
  box-shadow: 0px 1px 8px rgba(16, 24, 40, 0.06),0px 1px 10px rgba(16, 24, 40, 0.05);
  color: #EFEFEF;
  display: none;
  max-height: 48px;
  max-width: 48px;
  padding: 0;
  position: absolute;
  top: 32px;
}

.emotion-0 .footer__scroll-top .MuiFab-root .MuiSvgIcon-root {
  font-size: 40px;
}

.emotion-0 .footer__scroll-top .MuiFab-root:hover {
  background-color: #0939B0;
}

@media (min-width:375px) {
  .layout__scrolled .emotion-0 .footer__scroll-top .MuiFab-root {
    display: block;
  }
}

@media (min-width:1280px) {
  .emotion-0 .footer__scroll-top {
    background-image: url(/images/background/footerLg.svg);
    -webkit-background-position: 50% 20px;
    background-position: 50% 20px;
    background-repeat: no-repeat;
    min-height: 315px;
    padding: 10px 0 20px 0;
  }
}

@media (min-width:768px) {
  .emotion-0 .footer__scroll-top {
    background-image: url(/images/background/footerMd.svg);
    -webkit-background-position: 50% 20px;
    background-position: 50% 20px;
    background-repeat: no-repeat;
    min-height: 315px;
    padding: 10px 0 20px 0;
  }
}

@media (min-width:375px) {
  .emotion-0 .footer__scroll-top {
    background-image: url(/images/background/footerSm.svg);
    -webkit-background-position: 50% 20px;
    background-position: 50% 20px;
    background-repeat: no-repeat;
    min-height: 315px;
    padding: 10px 0 20px 0;
  }
}

@media (min-width:768px) {
  .emotion-0 .footer__social {
    margin: 0px 12px;
  }

  .emotion-0 .footer__social:first-of-type {
    margin-left: 0;
  }
}

.emotion-0 .footer__terms-dash {
  display: none;
  margin: 0px 8px;
}

.emotion-0 .footer__terms-dash:first-of-type {
  display: inline;
}

@media (min-width:768px) {
  .emotion-0 .footer__terms-dash {
    display: inline;
  }

  .emotion-0 .footer__terms-dash.footer__desktop {
    display: inline;
  }
}

.emotion-0 .footer__terms-links {
  margin: 8px 0px;
}

.emotion-0 .footer__terms-links .MuiTypography-root {
  font-size: 14px;
}

.emotion-0 .footer__terms-wrapper {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-box-flex-wrap: nowrap;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  min-width: 300px;
  padding: 0px 8px;
  width: 100%;
}

@media (min-width:768px) {
  .emotion-0 .footer__terms-wrapper {
    -webkit-align-items: baseline;
    -webkit-box-align: baseline;
    -ms-flex-align: baseline;
    align-items: baseline;
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
    -webkit-box-flex-flow: row wrap;
    -webkit-flex-flow: row wrap;
    -ms-flex-flow: row wrap;
    flex-flow: row wrap;
    -webkit-box-pack: end;
    -ms-flex-pack: end;
    -webkit-justify-content: flex-end;
    justify-content: flex-end;
    width: auto;
  }
}

.emotion-0 .MuiTypography-caption {
  color: #fff;
  font-size: 14px;
  font-weight: 500;
  line-height: 0.83333;
}

.emotion-0 .MuiTypography-caption a {
  font-size: 14px;
  line-height: 0.83333;
}

.emotion-0 .sectionlimit__container {
  -webkit-box-pack: center!important;
  -ms-flex-pack: center!important;
  -webkit-justify-content: center!important;
  justify-content: center!important;
  max-width: 100vw;
  min-width: 300px;
  padding: 8px 0px 0px 0px;
}

@media (min-width:768px) {
  .emotion-0 .sectionlimit__container {
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: justify;
    -webkit-justify-content: space-between;
    justify-content: space-between;
  }
}

.emotion-1 {
  margin-left: auto;
  margin-right: auto;
  max-width: 1024px;
}

.emotion-3 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-size: 0.875rem;
  font-style: normal;
  font-weight: 600;
  line-height: 1.214;
  font-family: 'SF Pro Text',-apple-system,BlinkMacSystemFont,sans-serif;
  text-transform: uppercase;
  min-height: 36px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border-radius: 50%;
  padding: 0;
  min-width: 0;
  width: 56px;
  height: 56px;
  z-index: 1050;
  box-shadow: 0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12);
  color: rgba(0, 0, 0, 0.87);
  background-color: #DDDDDD;
}

.emotion-3::-moz-focus-inner {
  border-style: none;
}

.emotion-3.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-3 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-3:active {
  box-shadow: 0px 7px 8px -4px rgba(0,0,0,0.2),0px 12px 17px 2px rgba(0,0,0,0.14),0px 5px 22px 4px rgba(0,0,0,0.12);
}

.emotion-3:hover {
  background-color: #f5f5f5;
  -webkit-text-decoration: none;
  text-decoration: none;
}

@media (hover: none) {
  .emotion-3:hover {
    background-color: #DDDDDD;
  }
}

.emotion-3.Mui-focusVisible {
  box-shadow: 0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12);
}

.emotion-3.Mui-disabled {
  color: rgba(255, 255, 255, 0.3);
  box-shadow: none;
  background-color: rgba(255, 255, 255, 0.12);
}

.emotion-4 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
}

.emotion-5 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-7 {
  margin: 0;
  font-size: 0.75rem;
  font-family: 'SF Pro Text',-apple-system,BlinkMacSystemFont,sans-serif;
  font-weight: 400;
  line-height: 1.66;
}

<div>
  <footer
    class="footer__container emotion-0"
  >
    <div
      class="sectionlimit__container MuiBox-root emotion-1"
    >
      <div
        class="footer__scroll-top MuiBox-root emotion-2"
      >
        <button
          class="MuiButtonBase-root MuiFab-root MuiFab-circular MuiFab-sizeLarge MuiFab-default MuiFab-root MuiFab-circular MuiFab-sizeLarge MuiFab-default emotion-3"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-4"
            data-testid="KeyboardArrowUpIcon"
            focusable="false"
            height="36"
            viewBox="0 0 24 24"
            width="36"
          >
            <path
              d="M7.41 15.41 12 10.83l4.59 4.58L18 14l-6-6-6 6z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root emotion-5"
          />
        </button>
        <div
          class="footer__bottom MuiBox-root emotion-2"
        >
          <div
            class="footer__icons-wrapper"
          >
            <a
              class="footer__social twitter"
              href="https://twitter.com/bigdipperlive"
              rel="noreferrer"
              target="_blank"
            >
              <svg
                fill="none"
                height="12"
                viewBox="0 0 14 12"
                width="14"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M12.33 3.23897C12.3353 3.35997 12.3353 3.47547 12.3353 3.59647C12.3406 7.26499 9.64538 11.5 4.71643 11.5C3.26269 11.5 1.83547 11.0655 0.609863 10.2515C0.822089 10.279 1.03431 10.29 1.24654 10.29C2.45092 10.29 3.62347 9.87199 4.57318 9.09649C3.42716 9.07449 2.41909 8.29899 2.06892 7.16599C2.47215 7.24849 2.88599 7.23199 3.2786 7.11649C2.03178 6.86349 1.13512 5.72498 1.12982 4.39948C1.12982 4.38848 1.12982 4.37748 1.12982 4.36648C1.50121 4.58098 1.92036 4.70198 2.34481 4.71298C1.17226 3.89898 0.806172 2.27647 1.51713 1.00597C2.88068 2.74397 4.88621 3.79448 7.04031 3.90998C6.82277 2.94747 7.11989 1.93547 7.81493 1.25347C8.89198 0.202964 10.5898 0.257964 11.6085 1.37447C12.208 1.25347 12.7863 1.02247 13.3116 0.697965C13.11 1.34147 12.6908 1.88597 12.1337 2.23247C12.6643 2.16647 13.1842 2.01797 13.6724 1.79797C13.3116 2.35897 12.8553 2.84297 12.33 3.23897Z"
                  fill="#FCCDCC"
                />
              </svg>
            </a>
            <a
              class="footer__social telegram"
              href="https://t.me/forbole"
              rel="noreferrer"
              target="_blank"
            >
              <svg
                fill="none"
                height="12"
                viewBox="0 0 16 12"
                width="16"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M1.89631 5.16592C5.78325 3.47245 8.37516 2.356 9.67202 1.81659C13.3748 0.276465 14.1442 0.00892841 14.6457 9.41924e-05C14.756 -0.00184879 15.0027 0.0254864 15.1624 0.155112C15.2973 0.264566 15.3344 0.412421 15.3522 0.516195C15.3699 0.619969 15.392 0.856369 15.3745 1.04108C15.1738 3.1494 14.3056 8.26572 13.8639 10.6271C13.677 11.6262 13.3089 11.9613 12.9527 11.9941C12.1784 12.0653 11.5904 11.4823 10.8405 10.9908C9.66695 10.2215 9.00399 9.74265 7.86489 8.992C6.54847 8.1245 7.40185 7.64771 8.15208 6.86849C8.34841 6.66457 11.76 3.56151 11.826 3.28001C11.8343 3.2448 11.8419 3.11357 11.764 3.04428C11.686 2.97498 11.5709 2.99868 11.4879 3.01753C11.3702 3.04424 9.4955 4.28334 5.86381 6.73483C5.33168 7.10023 4.8497 7.27826 4.41786 7.26893C3.94179 7.25865 3.02602 6.99976 2.34524 6.77846C1.51024 6.50704 0.846598 6.36353 0.904387 5.90257C0.934487 5.66247 1.26513 5.41692 1.89631 5.16592Z"
                  fill="#FCCDCC"
                  fill-rule="evenodd"
                  id="Vector"
                />
              </svg>
            </a>
            <a
              class="footer__social github"
              href="https://github.com/forbole"
              rel="noreferrer"
              target="_blank"
            >
              <svg
                fill="none"
                height="16"
                viewBox="0 0 18 16"
                width="18"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M9.1384 0C7.18005 0.00101608 5.28592 0.694626 3.79469 1.9568C2.30346 3.21898 1.31238 4.96742 0.998662 6.88949C0.684944 8.81155 1.06904 10.7819 2.08229 12.4482C3.09553 14.1145 4.67184 15.3682 6.52935 15.9849C6.93915 16.061 7.09351 15.807 7.09351 15.5911C7.09351 15.3751 7.08531 14.749 7.08258 14.0644C4.78771 14.5602 4.30278 13.096 4.30278 13.096C3.9285 12.1452 3.38757 11.8953 3.38757 11.8953C2.639 11.3873 3.44358 11.3969 3.44358 11.3969C4.27273 11.4553 4.70849 12.243 4.70849 12.243C5.44339 13.4967 6.63863 13.134 7.10853 12.9221C7.18229 12.3911 7.39676 12.0298 7.63308 11.8247C5.79991 11.6182 3.87386 10.9147 3.87386 7.77176C3.8625 6.95666 4.1667 6.1684 4.72351 5.57008C4.63882 5.36363 4.35605 4.52968 4.8041 3.39692C4.8041 3.39692 5.49667 3.17689 7.07302 4.23766C8.42512 3.87001 9.85169 3.87001 11.2038 4.23766C12.7788 3.17689 13.47 3.39692 13.47 3.39692C13.9194 4.52696 13.6366 5.36091 13.5519 5.57008C14.1105 6.1685 14.4153 6.95813 14.4029 7.77447C14.4029 10.9242 12.4728 11.6182 10.6369 11.8206C10.9319 12.076 11.1956 12.5744 11.1956 13.3405C11.1956 14.4379 11.186 15.3208 11.186 15.5911C11.186 15.8097 11.3349 16.0651 11.7529 15.9849C13.6106 15.3681 15.1871 14.1142 16.2003 12.4476C17.2136 10.781 17.5975 8.81039 17.2834 6.88815C16.9693 4.9659 15.9777 3.21745 14.486 1.95549C12.9943 0.693525 11.0997 0.000372037 9.14113 0H9.1384Z"
                  fill="#FCCDCC"
                  fill-rule="evenodd"
                />
              </svg>
            </a>
            <a
              class="footer__social linkedin"
              href="https://www.linkedin.com/company/forbole"
              rel="noreferrer"
              target="_blank"
            >
              <svg
                fill="none"
                height="14"
                viewBox="0 0 15 14"
                width="15"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M13.1076 0H1.17459C0.900492 0 0.637624 0.108884 0.44381 0.302697C0.249997 0.496511 0.141113 0.759378 0.141113 1.03347V12.9665C0.141113 13.2406 0.249997 13.5035 0.44381 13.6973C0.637624 13.8911 0.900492 14 1.17459 14H13.1076C13.3817 14 13.6446 13.8911 13.8384 13.6973C14.0322 13.5035 14.1411 13.2406 14.1411 12.9665V1.03347C14.1411 0.759378 14.0322 0.496511 13.8384 0.302697C13.6446 0.108884 13.3817 0 13.1076 0ZM4.31389 11.9262H2.20903V5.24028H4.31389V11.9262ZM3.26 4.31375C3.02124 4.31241 2.78823 4.24036 2.59037 4.10672C2.39252 3.97307 2.23869 3.78381 2.14829 3.56282C2.0579 3.34183 2.03499 3.09901 2.08246 2.86502C2.12994 2.63102 2.24566 2.41633 2.41504 2.24804C2.58441 2.07975 2.79984 1.96541 3.03414 1.91944C3.26843 1.87346 3.5111 1.89793 3.7315 1.98974C3.95191 2.08155 4.14018 2.2366 4.27255 2.43531C4.40492 2.63402 4.47547 2.86749 4.47528 3.10625C4.47753 3.2661 4.44758 3.42477 4.3872 3.5728C4.32683 3.72083 4.23727 3.85519 4.12386 3.96787C4.01046 4.08055 3.87552 4.16925 3.72711 4.22867C3.5787 4.28809 3.41984 4.31703 3.26 4.31375ZM12.0722 11.9321H9.96834V8.27944C9.96834 7.20222 9.51042 6.86972 8.91931 6.86972C8.29514 6.86972 7.68264 7.34028 7.68264 8.30667V11.9321H5.57778V5.24514H7.60195V6.17167H7.62917C7.83236 5.76042 8.54403 5.0575 9.63 5.0575C10.8044 5.0575 12.0732 5.75458 12.0732 7.79625L12.0722 11.9321Z"
                  fill="#FCCDCC"
                />
              </svg>
            </a>
          </div>
          <div
            class="footer__terms-wrapper"
          >
            <span
              class="MuiTypography-root MuiTypography-caption emotion-7"
            >
              copyright
            </span>
            <span
              class="MuiTypography-root MuiTypography-caption footer__terms-dash footer__desktop emotion-7"
            >
              |
            </span>
            <div
              class="footer__terms-links"
            >
              <a
                href="/terms-and-conditions"
                target="_self"
              >
                <span
                  class="MuiTypography-root MuiTypography-caption footer__link-item footer__link-item-caption emotion-7"
                >
                  termsAndConditions
                </span>
              </a>
              <span
                class="MuiTypography-root MuiTypography-caption footer__terms-dash emotion-7"
              >
                |
              </span>
              <a
                href="/privacy-policy"
                target="_self"
              >
                <span
                  class="MuiTypography-root MuiTypography-caption footer__link-item footer__link-item-caption emotion-7"
                >
                  privacyPolicy
                </span>
              </a>
            </div>
            <span
              class="MuiTypography-root MuiTypography-caption footer__terms-dash footer__desktop emotion-7"
            >
              |
            </span>
            <span
              class="MuiTypography-root MuiTypography-caption emotion-7"
            >
              common:productOf
            </span>
          </div>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
